e107_plugins/forum/shortcodes/batch/post_shortcodes.php

```diff
@@ -434,6 +434,29 @@ function sc_forum_post_breadcrumb()
 		return $_tmp['breadcrumb'];
 	}
 
+	function sc_forum_post_captionname()
+	{
+//		global $forumInfo;
+			$tp = e107::getParser();
+//    var_dump ($this);
+//$this->forumObj->threadGet($this->id, false)		
+    if ($this->var['action'] == "rp")
+    {
+      $pre = LAN_FORUM_1003;
+			$name = strip_tags($tp->toHTML($this->var['thread_name'], false, 'no_hook, emotes_off'));
+			$url = e107::url('forum', 'topic', $this->var);
+      $post = LAN_FORUM_2006;
+    }
+    if ($this->var['action'] == "nt")
+    {
+      $pre = LAN_FORUM_1001;
+			$name = strip_tags($tp->toHTML($this->var['forum_name'], false, 'no_hook, emotes_off'));
+			$url = e107::url('forum', 'forum', $this->var);
+      $post = LAN_FORUM_2005;
+    }
+    return $pre.($url?": <a {$title} href='".$url."'>{$name}</a> - ":$name).$post;
+	}
+	
```

Comment:
I was trying to get rid of the HTML on the variable, but anyway, i can live with it....